src/crypto-providers/ledgerCryptoProvider.ts

```diff
@@ -106,21 +122,26 @@ export const LedgerCryptoProvider: () => Promise<CryptoProvider> = async () => {
     }
   }
 
-  const determineSigningMode = (certificates: _Certificate[], signingFiles: HwSigningData[]) => {
-    const poolRegistrationCert = certificates.find(
+  const determineSigningMode = (
+    unsignedTxParsed: _UnsignedTxParsed,
```

Comment:
Some of these issues will be resolved by integrating interop-lib and using its types (eg. `RawTransaction` instead of `_UnsignedTxParsed`).

Reply:
I'll let @davidmisiak decide whether all the problems can be resolved during the library integration, and if not, he can create a github issue to deal with any remaining refactors later.

Reply:
It seems to me that this will be resolved - after interop-lib integration, there will be the CLI arguments types of the form `ParsedXxxArguments` and the tx types from interop-lib.